pkgs/development/tools/misc/dlang-dfmt/mkDub.nix

```
@@ -0,0 +1,141 @@
+{ pkgs ? import <nixpkgs> { }, stdenv ? pkgs.stdenv, lib ? pkgs.lib
+, dtools ? pkgs.dtools or pkgs.rdmd, dmd ? pkgs.dmd, dcompiler ? dmd
+, dub ? pkgs.dub }:
```

`pkgs` should almost never be used as an argument to functions which get called with `callPackage`. Let's remove it.

I've also intercalated the arguments with newlines to make it easier to review the diffs in the future, when people introduce or remove arguments.

```suggestion
{ lib
, stdenv
, dtools
, dmd
, dcompiler ? dmd
, dub
}:
```